cypress/integration/account_settings/display/channel_display_mode_spec.js

```diff
@@ -120,15 +124,15 @@ describe('Account Settings > Display > Channel Display Mode', () => {
 
     it('Width of center view when message display is compact and channel display mode is either centered or full', () => {
         // 1. Set message display to compact with channel display mode set to centered
-        cy.updateMessageDisplayPreference('compact');
+        cy.apiSaveMessageDisplayPreference('compact');
 
         // * Verify that the width is at 1000px
         cy.get('.post__content').last().should('have.css', 'width', '1000px');
 
         // 2. Set channel display mode to full (default) with message display in compact
-        cy.updateChannelDisplayModePreference();
+        cy.apiSaveChannelDisplayModePreference();
 
-        // * Verify that the width is at 1123px
-        cy.get('.post__content').last().should('have.css', 'width', '1123px');
+        // * Verify that the width is at 1131px
+        cy.get('.post__content').last().should('have.css', 'width', '1131px');
```

Comment:
Not related but fix anyway. I'm not particularly sure what has changed but there might be a bit of change in styles recently.

Reply:
hmm, should we change this to use closeTo assertion instead of exact comparison?

Reply:
What I did is just compare the exact classname to `#postContent`.  Please take a look again.  Also in the latest commit, I removed the unnecessary test.

Reply:
It looks much better now :+1: